intranet/apps/emerg/views.py

```diff
@@ -66,7 +66,6 @@ def check_emerg():
 
 def get_emerg_result():
     """Run the fetch command from FCPS."""
-    logger.debug("Fetching emergency info from FCPS")
```

Comment:
I would actually keep this. It might be helpful to know when we last fetched emergency info from FCPS.